packages/account/src/Components/poi/idv-status/idv-failed/idv-failed.tsx

```diff
@@ -361,79 +372,77 @@ const IdvFailed = ({
         return '80px';
     };
 
+    const buttonText = (is_idv_skipping: boolean) =>
+        is_idv_skipping
+            ? localize('Next')
+            : is_document_upload_required
+            ? localize('Verify')
+            : localize('Update profile');
+
```

Comment:
This nested ternary operation will have code smells in sonarcloud.

Reply:
yes, will refactor this function 👍